community/contributing/pr_workflow.rst

```diff
@@ -351,6 +351,10 @@ Git will ask you for your username and password, and the changes will be sent
 to your remote. If you check the fork's page on GitHub, you should see a new
 branch with your added commits.
 
+.. note:: Since july 2020, GitHub changed the way you verify yourself in HTTPS clones.
+          The password you must introduce is a `PAT
+https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token`_
```

Comment:
Doesn't GitHub send you a message when trying to push to an HTTPS remote to warn you about this? I feel we should avoid duplicating their documentation.

Reply:
Absolutely right, should i revert it to keep the other commit?

Reply:
@DiegoCCMM Yes, I think it's better to revert this part for now.